plugins/execute/transformerExporter.go

```diff
@@ -132,126 +133,127 @@ var Exporter exporter
+        log_median_price.EventTransformerInitializer,
+        median_lift.EventTransformerInitializer,
+        vat_deny.EventTransformerInitializer,
+        tend.EventTransformerInitializer,
+        vat_hope.EventTransformerInitializer,
+        flip_file_cat.EventTransformerInitializer,
+        spot_poke.EventTransformerInitializer,
+        jug_file_ilk.EventTransformerInitializer,
+        pot_file_vow.EventTransformerInitializer,
+        vow_heal.EventTransformerInitializer,
+        vat_frob.EventTransformerInitializer,
+        jug_file_base.EventTransformerInitializer,
+        deal.EventTransformerInitializer,
+        new_cdp.EventTransformerInitializer,
+        pot_join.EventTransformerInitializer,
+        pot_exit.EventTransformerInitializer,
+        jug_init.EventTransformerInitializer,
+        vow_flog.EventTransformerInitializer,
+        rely.EventTransformerInitializer,
+        dent.EventTransformerInitializer,
+        vow_file_auction_attributes.EventTransformerInitializer,
+        flip_kick.EventTransformerInitializer,
+        vat_slip.EventTransformerInitializer,
+        spot_file_mat.EventTransformerInitializer,
+        log_value.EventTransformerInitializer,
+        cat_claw.EventTransformerInitializer,
+        bite.EventTransformerInitializer,
+        auction_file.EventTransformerInitializer,
+        jug_file_vow.EventTransformerInitializer,
+        vat_fork.EventTransformerInitializer,
+        vat_init.EventTransformerInitializer,
+        flop_kick.EventTransformerInitializer,
+        pot_drip.EventTransformerInitializer,
+        spot_file_par.EventTransformerInitializer,
+        vow_file_auction_address.EventTransformerInitializer,
+        vat_heal.EventTransformerInitializer,
+        deny.EventTransformerInitializer,
+        vat_fold.EventTransformerInitializer,
+        vat_file_debt_ceiling.EventTransformerInitializer,
+        vat_suck.EventTransformerInitializer,
+        median_drop.EventTransformerInitializer,
+        cat_file_flip.EventTransformerInitializer,
+        cat_file_chop_lump_dunk.EventTransformerInitializer,
+        yank.EventTransformerInitializer,
+        vow_fess.EventTransformerInitializer,
+        vat_nope.EventTransformerInitializer,
+        median_diss_batch.EventTransformerInitializer,
+        pot_file_dsr.EventTransformerInitializer,
+        median_kiss_batch.EventTransformerInitializer,
+        flap_kick.EventTransformerInitializer,
+        vat_grab.EventTransformerInitializer,
+        median_diss_single.EventTransformerInitializer,
+        pot_cage.EventTransformerInitializer,
+        cat_file_vow.EventTransformerInitializer,
+        vat_flux.EventTransformerInitializer,
+        vat_rely.EventTransformerInitializer,
+        vat_move.EventTransformerInitializer,
+        spot_file_pip.EventTransformerInitializer,
+        vat_file_ilk.EventTransformerInitializer,
+        osm_change.EventTransformerInitializer,
```

Comment:
could be cool to alphabetize and format this file, if only to make the diff clearer (added/removed transformers)